lib/matplotlib/tests/test_backend_qt.py

```diff
@@ -295,3 +311,25 @@ def crashing_callback(fig, stale):
     canvas = FigureCanvasQTAgg(fig)
     fig.stale = True
     assert called
+
+
+@pytest.mark.backend('Qt5Agg', skip_on_importerror=True)
+def test_form_widget_get_with_datetime_field():
+    if not QtWidgets.QApplication.instance():
+        QtWidgets.QApplication()
+    form = [("Field name", datetime(year=2021, month=3, day=11))]
+    widget = _formlayout.FormWidget(form)
+    widget.setup()
+    values = widget.get()
+    assert(values == [datetime(year=2021, month=3, day=11)])
+
+
+@pytest.mark.backend('Qt5Agg', skip_on_importerror=True)
+def test_form_widget_get_with_date_field():
```

Comment:
I would merge this test with the one above to avoid paying twice for the setup of the widget.

Reply:
Done.